docs/combat-mechanics/turn-order/turn-order-mechanics.md

```diff
@@ -0,0 +1,111 @@
+---
+description: An explanation on how turn order is determined during combat.
+---
+
+# Turn Order Mechanics
+
+## Action Gauge
+
+Starts at 10000 by default for all units, once it reaches 0 for any given unit it is their turn to act. 
+
```

Comment:
nit: would be nice to have this also explain the starting order + what happens in natural ties:
1. all units are added to the order in the order of char 1 -> 4, enemy 1 -> 4
2. stable sort of this order by AV (in the event of ties, all chars go before any enemies, char 1 goes before char 2, char 2 before 3, char 3 before 4)
3. when a turn reset happens (end of action), the unit gauge gets reset to 0 and they get put back in the order. In the event of a tie, they will be at the bottom of the tied set